src/packages/navbar/navbar.harmony.css

```
@@ -0,0 +1,106 @@
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+}
+.nut-navbar-placeholder {
+  display: inline-block;
+  width: 100%;
+}
+.nut-navbar-safe-area-inset-top {
+  padding-top: constant(safe-area-inset-top);
```

Biome (1.9.4)

[error] Unexpected unknown function: constant

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)